tarorpc/taro.proto

```diff
@@ -87,6 +87,11 @@ service Taro {
     proof file information the receiver needs to fully receive the asset.
     */
     rpc SendAsset (SendAssetRequest) returns (SendAssetResponse);
+
+    /* tarocli: `getinfo`
+    GetInfo returns the information for the node.
```

Comment:
nit: "GetInfo returns information about the daemon" in order to match the CLI text?